src/player.py

```diff
@@ -1,2 +1,14 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+
+    def __init__(self, name, current_room):
+        self.name = name
+        self.hp = 100
```

Comment:
I'm glad you know that you can set attributes without having them passed in. Good work!